packages/react/src/use-render/useRender.ts

```diff
@@ -20,7 +20,7 @@ export function useRender<
   };
   renderParams.disableStyleHooks = true;
 
-  return useRenderElement(undefined, renderParams, renderParams);
+  return useRenderElement(renderParams.defaultTag, renderParams, renderParams);
```

Comment:
The terminology feels wrong. "tag" does not seem part of the React language. 

 If we follow https://legacy.reactjs.org/docs/implementation-notes.html#mounting-host-elements:

```suggestion
  return useRenderElement(renderParams.defaultHostElement, renderParams, renderParams);
```

If we follow: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/64de0973102a5a89ac3c5c4f9913459fd93cb273/types/react/index.d.ts#L4051

```suggestion
  return useRenderElement(renderParams.defaultIntrinsicElement, renderParams, renderParams);
```

Reply:
But I wonder, why only support host elements:
```suggestion
  return useRenderElement(renderParams.defaultComponent, renderParams, renderParams);
```

Reply:
Yeah I don't know the best word for this 😬  

I find the word "host" to be pretty abstract, and sort of implies knowledge of how React's internals work, while HTML "tag" is common knowledge. I  also see the word "element" representing the object value when writing JSX, not a string. `tag` matches `ref.current.tagName`, except in lowercase

Reply:
The `useRender` hook is specifically for building primitive components where you aren't rendering another component (yet). The callsite of the component that has `useRender` is where they do the replacing with `render`

Reply:
Makes sense. I'm ok with `defaultTag` in that case. The other terms, "intrinsic elements" and "host elements", aren't used enough outside of react's codebase for them to be easily understandeable. Anyway the typings on `defaultTag` will make it very clear what goes in there.

Reply:
> I wonder if we won't see cases where we need to expand the need to support any components.

And if we pick `defaultComponent` for this param name now, we could later expand the typings without a breaking change.

Reply:
I would use `defaultTag` as it's the only things we support now. If in the future we want to extend the behavior, we can add a new option `defaultComponent` and deprecate the `defaultTag` in favor of it. It would feel wrong naming it `defaultComponent` if it does not support custom components.

Reply:
> It would feel wrong naming it defaultComponent if it does not support custom components.

Agree